packages/world-module-metadata/ts/build.ts

```diff
@@ -0,0 +1,17 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { tablegen } from "@latticexyz/store/codegen";
+import { worldgen } from "@latticexyz/world/node";
+
+/**
+ * To avoid circular dependencies, we run a very similar `build` step as `cli` package here.
+ */
+
+// TODO: move tablegen/worldgen to CLI commands from store/world we can run in package.json instead of a custom script
```

Comment:
wanna open an issue for this? seems like a "good first issue"

Reply:
done: https://github.com/latticexyz/mud/issues/3030